packages/grpc-web/test/plugin_test.js

```diff
@@ -214,4 +216,38 @@ describe('grpc-web plugin test, proto with no package', function() {
     var myClient = new GreeterClient("MyHostname", null, null);
     assert.equal('function', typeof myClient.sayHello);
   });
+
+  it('PromiseClient: should exist', function() {
+    execSync(genCodeCmd);
+
+    const {HelloRequest} = require(genCodePath1);
+
+    const {GreeterPromiseClient} = require(genCodePath2);
+    var myClient = new GreeterPromiseClient("MyHostname", null, null);
+    assert.equal('function', typeof myClient.sayHello);
+
+    var p = myClient.sayHello(new HelloRequest(), {});
+    assert.equal('function', typeof p.then);
+  });
+
+  it('PromiseClient: response should resolve promise', function() {
+    execSync(genCodeCmd);
```

Comment:
Is it needed to run this on each test method or could it be run in the `beforeAll` method?

Same applies to`HelloRequest`, `GreeterPromiseClient`, etc.